data/json/vehicleparts/vehicle_parts.json

```diff
@@ -2631,6 +2631,22 @@
     "flags": [ "NOINSTALL", "UNMOUNT_ON_DAMAGE", "UNMOUNT_ON_MOVE", "POWER_TRANSFER" ],
     "breaks_into": [ { "item": "wire", "count": [ 4, 8 ] }, { "item": "plastic_chunk", "count": [ 4, 8 ] } ]
   },
+  {
+    "type": "vehicle_part",
+    "id": "tow_cable",
+    "name": "tow cable",
+    "symbol": "{",
+    "color": "light_blue",
+    "broken_symbol": "*",
+    "broken_color": "dark_gray",
+    "damage_modifier": 10,
+    "durability": 120,
+    "description": "A tow cable, if the other end was attached to another vehicle, it could pull it.",
+    "item": "tow_cable",
+    "requirements": { "removal": { "time": 500 } },
+    "flags": [ "NOINSTALL", "UNMOUNT_ON_DAMAGE", "TOW_CABLE" ],
+    "breaks_into": [ { "item": "wire", "count": [ 4, 8 ] }, { "item": "plastic_chunk", "count": [ 4, 8 ] } ]
```

Comment:
There is no wire or plastic in crafting recipe of this cable at all. Also I think grip hook could be dropped too.

Reply:
I meant add grip hook to the list of what drops when part is broken.

Reply:
Ohh lol, sure yes.